app/components/Reactions/Reaction.js

```diff
@@ -34,10 +35,14 @@ class Reaction extends React.Component<Props> {
       addReaction,
       deleteReaction,
       hasReacted,
+      canReact,
       reactionId,
       contentTarget,
     }: Props = this.props;
-    const classes = [className ? className : styles.reaction];
+    const classes = [
+      className ? className : styles.reaction,
+      ...(canReact ? [styles.clickable] : []),
```

Comment:
`classnames` can take false values https://www.npmjs.com/package/classnames, so you can actually don't need to pass the empty list if they cannot react. So you can actually do `canReact && styles.clickable`. No need for checks 👍

Reply:
```js
// other falsy values are just ignored
classNames(null, false, 'bar', undefined, 0, 1, { baz: null }, ''); // => 'bar 1'
```

Reply:
Ah, cool, i'll do that